### ISSUE TITLE
data file has two scan #1 but second is not recognized

### ISSUE THREAD

**Issue by prjemian** [prjemian/spec2nexus#39]
A certain data file (CdOsO) was re-opened with the ```newfile``` macro and the scan numbering was restarted from 1.  There are now two scans numbered ```#1``` but the second one cannot be found using spec2nexus.

Note that PyMCA can access this scan, calling it ```#1.2``` (the first one is ```#1.1```)

**Comment by prjemian** [comment]
On line ```#3691```, there is a ```#F``` at the start of the line.  This marks where the file has been re-opened for writing SPEC additional scans where the scan numbering starts from 1 (again).

I believe my assumption is that ```#F``` lines that are not at the start of the file will always have a blank line before them.  That assumption fails with this file.

Most SPEC data files only have a single ```#F``` line, on line 1.

**Comment by prjemian** [comment]
Actually, there are four ```#E``` lines (2, 3659, 3692, 3800) and those are identified properly.  Looks like the user was struggling with the ```newfile``` command.  A good test case for this pathology in a SPEC data file.

**Comment by prjemian** [comment]
in ```plugins/spec_commons_spec2nexus.py``` this code:

```
        if scan.scanNum in spec_obj.scans:
            msg = str(scan.scanNum) + ' in ' + spec_obj.fileName
            raise DuplicateSpecScanNumber(msg)
```

Instead of raising an exception, write as ```"%d.%d" % (scan.scanNum, index)``` where *index* is the lowest integer in ```range(2,really big)``` that is not already in use.

**Comment by prjemian** [comment]
fixed by #41
